scripts/collection_from_items.py

```diff
@@ -187,6 +194,9 @@ def main(args: List[str] | None = None) -> None:
         duration=time_in_ms() - start_time,
     )
 
+    if arguments.capture_dates:
+        collection.add_capture_dates(uri)
```

Comment:
I would do that earlier in the process (before adding items, capture-area loop) so if it fails earlier if the file does not exist (it's relying on an external file)